CHANGELOG.rst

```diff
@@ -1,5 +1,45 @@
 Changelog
 =========
+1.14.0 (2021-02-18)
+-------------------
+MLflow 1.14.0 includes several major features and improvements:
+
+Support for tensor inputs using the TF serving format:
+
+- Add support for TF serving input to REST API (#3894, @wentinghu)
+- Add support for np array-based tensors for Keras, PyTorch and TensorFlow flavors (#3808, @wentinghu)
+- Add tensor input support for ONNX and gluon flavors (#4041, @tomasatdatabricks)
+- Add support for tensor-based model signatures in schema enforcement (#4084, @wentinghu)
+
+For more information, see https://mlflow.org/docs/latest/models.html#deploy-mlflow-models
+
+More features and improvements:
+
+- Add SHAP Explainer logging integration (#3989, @vivekchettiar)
```

Comment:
```suggestion
- Added new `mlflow.shap.log_explainer`, `mlflow.shap.load_explainer` APIs for logging and loading shap.Explainer instances (#3989, @vivekchettiar)
```

In general when we've added new APIs or changed the behaviors of existing APIs, it's useful to mention the impacted APIs directly, so that users can look them up in the docs/try them out :)